sharktank/conftest.py

```diff
@@ -121,11 +121,34 @@ def pytest_addoption(parser):
     )
 
     parser.addoption(
-        "--baseline-perplexity-score-json",
+        "--baseline-perplexity-scores",
         type=Path,
         action="store",
-        default="sharktank/tests/evaluate/baseline_perplexity_scores.json",
-        help="Llama3.1 8B & 405B model baseline perplexity scores json",
+        default="/home/aramalin/SHARK-Platform/sharktank/tests/evaluate/baseline_perplexity_scores.json",
+        help="Llama3.1 8B & 405B model baseline perplexity scores",
+    )
+
+    parser.addoption(
+        "--llama3-8b-f16-vmfb-path",
+        type=Path,
+        action="store",
+        default="/data/extra/models/llama3.1_8B/llama8b_f16.vmfb",
+        help="Llama3.1 8b fp16 vmfb path, defaults to 30F CI system path",
+    )
+
+    parser.addoption(
+        "--llama3-405b-f16-vmfb-path",
+        type=Path,
+        action="store",
+        default="/data/extra/models/llama3.1_405B/llama405b_fp16.vmfb",
+        help="Llama3.1 405b fp16 vmfb path, defaults to 30F CI system path",
+    )
+
+    parser.addoption(
+        "--iree-device",
```

Comment:
Not clear to me what device we are running for the CI, where are we setting that?

Reply:
Will add `--device='hip://0'` flag in yaml to vmfb pytest. The vmfb is compiled with `--iree-hip-target=gfx942`, so it needs to run on MI300.